models/pathology_tumor_detection/configs/train.json

```diff
@@ -32,7 +32,7 @@
         "model_name": "resnet18",
         "num_classes": 1,
         "use_conv": true,
-        "pretrained": true
+        "pretrained": false
```

Comment:
I did this change according to @KumoLiu 's comment on:
https://github.com/Project-MONAI/model-zoo/pull/357
Hi @drbeh , may need your double confirm here.

Reply:
Thanks @yiheng-wang-nv, from development perspective, it doesn't matter and keeping it as `true` is fine (and more favorable for users). However, it is rather a product decision where we should see if we want to ship the pipelines to be able to be trained on ImageNet out-of-the-box or not, so I differ to @daguangxu and @pdogra89 for that.

PS: What is the situation with other pipeline? Are they using pretrained weights by default?

Reply:
Hi @drbeh ,

For all the bundles which can use pretrained weights, I think we should set `pretrained=False`, because:
1. ImageNet is not for commercial use.
2. Our `model.pt` was trained by private dataset pretrained weights.

Thanks.

Reply:
I also do not think it is critical. Either way is fine. Since all current bundle use False. Let's keep it simple here.